mmaction/datasets/pipelines/loading.py

```diff
@@ -53,7 +48,6 @@ def __init__(self,
         self.clip_len = clip_len
         self.frame_interval = frame_interval
         self.num_clips = num_clips
-        self.start_index = start_index
```

Comment:
for easier migration for users, we may check the use of start_index here and raise a detailed error message, describing how to modify code (or link to a faq section of docs)

Reply:
It is hard to check `start_index` here, since we can not get the result dict here to judge whether it's a video or rawframe. Besides, For videos the `start_index` should be 0, but for rawframes it depends on the files themselves to assign 0 or 1 to `start_idx`.

Reply:
Check it here, in the __init__

because we no longer support start_index in init, any usage of it should be warned